### ISSUE TITLE
Re-enable tests disabled as part of repo merge

### ISSUE THREAD

**Issue by dsplaisted** [dotnet/sdk#3684]
I'm working on merging the cli and toolset repos, and there are some tests which are failing only on non-Windows and only in CI.  I am going to disable these tests for now in order to not block the repo merge.

Once everything is merged into the SDK repo, we should investigate and re-enable them.

**Comment by dsplaisted** [comment]
I'm disabling a bunch of the ToolPackage tests against this bug.  The issue is that they end up shelling out to a dotnet process that uses stage 0 dotnet.exe to try to restore a project with the stage 2 layout.

**Comment by wli3** [comment]
there is no stage 2 layout, it is dotnet.dll and dependencies. Shell out is how the result of how command resolver is implemented.

**Comment by wli3** [comment]
The tests are testing the nuget adapter (giving package name, output location, download a package.)

by the time nuget is from LKG anyway, so stage 0 or stage 2 they are the same. Now stage 2 has the latest nuget, the test should run again stage2.

I can swap the implementation of IProjectRestorer, and let it use stage 2

**Comment by dsplaisted** [comment]
Re-opening this, as there are still tests disabled against this for other reasons.
